config/metrics.js

```diff
@@ -61,6 +61,11 @@ module.exports = {
 		extract: (item) => calcRestrictedSourcePercentage(getBPRules(item)),
 		permissions: ['ADMIN'],
 	},
+	RequiredDeploymentEnvironments: {
+		extract: (item) => !!getBPRules(item)?.requiredDeploymentEnvironments,
```

Comment:
```suggestion
		extract: (item) => !!getBPRules(item)?.requiredDeploymentEnvironments?.length > 0,
```
i assume? since an empty array means nothing's required?